sbt/src/sbt-test/project/unified/build.sbt

```diff
@@ -0,0 +1,12 @@
+lazy val root = (project in file("."))
+  .settings(
+    Global / cancelable := true,
+    ThisBuild / scalaVersion := "2.12.2",
```

Comment:
> We can provide out-of-the-box syntactic rewrites so that users don't worry about this

How long do you think it would take for this to be available? It would be good to test this on a lot of projects before RC1 (assuming that RC1 means feature freeze, regression fixes only).